### ISSUE TITLE
FEATURE: Keyboard shortcut for Search

### ISSUE THREAD

**Issue by nreliu** [Musish/Musish#368]
I'd love to be able to bring up the search input field with a keyboard shortcut.

**Comment by jamesjarvis** [comment]
Sounds like a great idea! If you want to say you contributed to the worlds best open source Apple Music web player - submit a PR for it! 😄 I'd suggest typing "s" or something should bring up the search box?

**Comment by jeewonkoo** [comment]
Hey, I would like to work on this issue. Can you provide me detailed information?

**Comment by AlexanderSutul** [comment]
@jeewonkoo  I'd support if you need some help.

**Comment by jeewonkoo** [comment]
@AlexanderSutul Sounds good! However, I am not sure if this repo is actively maintained?

**Comment by AlexanderSutul** [comment]
> @AlexanderSutul Sounds good! However, I am not sure if this repo is actively maintained?

Yeah, I think you're right devs are not really maintain this repo actively. But this app is good (My personal opinion). And it could be a good to improve it. Maybe it will also increase amount of supporters 😊

**Comment by jeewonkoo** [comment]
> > @AlexanderSutul Sounds good! However, I am not sure if this repo is actively maintained?
> 
> 
> 
> Yeah, I think you're right devs are not really maintain this repo actively. But this app is good (My personal opinion). And it could be a good to improve it. Maybe it will also increase amount of supporters 😊  

Agreed. Let's work on it then!

**Comment by tyrkinn** [comment]
@jamesjarvis I can work on it. Maybe cmd+k would be nice shortcut?

**Comment by jamesjarvis** [comment]
> [@jamesjarvis](https://github.com/jamesjarvis) I can work on it. Maybe cmd+k would be nice shortcut?

cmd+k or '/' would both be great!

**Comment by heyimteee** [comment]
is this still open? i would like to work on this!
